src/probnum/randprocs/markov/continuous/_lti_sde.py

```diff
@@ -0,0 +1,223 @@
+"""LTI SDE models as transitions."""
+import functools
+
+import numpy as np
+
+from probnum.randprocs.markov import discrete
+from probnum.randprocs.markov.continuous import _linear_sde, _mfd
+
+
+class LTISDE(_linear_sde.LinearSDE):
+    """Linear time-invariant continuous Markov models of the form.
+
+    .. math:: d x(t) = [G x(t) + v] d t + L d w(t).
+
+    In the language of dynamic models,
+
+    x(t) : state process
+    G : drift matrix
+    v : force term/vector
+    L : dispersion matrix.
+    w(t) : Wiener process with unit diffusion.
+
+    Parameters
+    ----------
+    drift_matrix :
+        This is F. It is the drift matrix of the SDE.
+    force_vector :
+        This is U. It is the force vector of the SDE.
+    dispersion_matrix :
+        This is L. It is the dispersion matrix of the SDE.
+    """
+
+    def __init__(
+        self,
+        drift_matrix: np.ndarray,
+        force_vector: np.ndarray,
+        dispersion_matrix: np.ndarray,
+        forward_implementation="classic",
+        backward_implementation="classic",
+    ):
+        # Assert all shapes match
+        _check_initial_state_dimensions(drift_matrix, force_vector, dispersion_matrix)
+
+        # Convert input into super() compatible format and initialize super()
+        state_dimension = drift_matrix.shape[0]
+        wiener_process_dimension = dispersion_matrix.shape[1]
+
+        def drift_matrix_function(t):
+            return drift_matrix
+
+        def force_vector_function(t):
+            return force_vector
+
+        def dispersion_matrix_function(t):
+            return dispersion_matrix
+
+        super().__init__(
+            state_dimension=state_dimension,
+            wiener_process_dimension=wiener_process_dimension,
+            drift_matrix_function=drift_matrix_function,
+            dispersion_matrix_function=dispersion_matrix_function,
+            force_vector_function=force_vector_function,
+        )
+
+        # Initialize remaining attributes
+        self._drift_matrix = drift_matrix
+        self._force_vector = force_vector
+        self._dispersion_matrix = dispersion_matrix
+        self._forward_implementation_string = forward_implementation
+        self._backward_implementation_string = backward_implementation
+
+    @property
+    def drift_matrix(self):
+        return self._drift_matrix
+
+    @property
+    def force_vector(self):
+        return self._force_vector
+
+    @property
+    def dispersion_matrix(self):
+        return self._dispersion_matrix
+
+    @property
+    def forward_implementation(self):
+        return self._forward_implementation_string
+
+    @property
+    def backward_implementation(self):
+        return self._backward_implementation_string
```

Comment:
before, it was possible to overwrite `self.drift_matrix` with some potentially unexpected behaviour (i.e. the wrappers that initialise superclasses as in `nonlinear(t, x) = linear(t) @ x` might not have been changed. This is impossible now, and the only way of changing the object is via `self.duplicate(...)`.

Reply:
Yep, I like the introduction of `@property`s. We could also consider defining a `setter` and throwing a respective warning that the user is then up to him/herself. :D But I wouldn't.
 
Super minor: I just wonder whether we should rename the `forward`-/`backward_implementation` to `..._string` as well. The name otherwise suggests some kind of functionality (resp. some kind of `implementation`).

Reply:
Yes, a setter could work, though I agree with you that this would be a bit pointless. 

The string: yes, that seems like a good idea to me.